CHANGELOG.md

```diff
@@ -10,10 +10,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Added
 
-- Add template attribute functions to the `go.opentelmetry.io/otel/semconv/v1.34.0` package. (#6939)
+- Add template attribute functions to the `go.opentelmetry.io/otel/semconv/v1.34.0` package. (#6939, #6962)
   - `ContainerLabel`
   - `DBOperationParameter`
   - `DBSystemParameter`
+  - `ErrorType`
```

Comment:
`ErrorType` is not a template attribute function. This should be its own changelog entry describing the specific functionality it is adding.